WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -42,9 +42,10 @@ final class ProductsViewController: UIViewController {
 
     /// Top toolbar that shows the sort and filter CTAs.
     ///
-    private lazy var toolbar: UIView = {
-        return createToolbar()
-    }()
+    @IBOutlet weak var toolbar: ToolbarView!
+
+    // Used to trick the navigation bar for large title (ref: issue 3 in p91TBi-45c-p2).
+    private let hiddenScrollView = UIScrollView()
```

Comment:
ah, unfortunately we need this workaround for the products tab now 😞 wish Apple had a better handling and API for large titles

Reply:
Yeah, we need it as we did for orders. Fortunately, I found your p2 with the solution and the explanation because I spent a while wondering what was happening, thanks for that 😅